generators/typescript/sdk/CHANGELOG.md

```diff
@@ -5,16 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.39.2] - 2024-08-16
+
+- Fix: Allows filenames to be passed from underlying File objects in Node 18+ and browsers
```

Comment:
@RohinBhargava can you add a code snippet of how someone would pass in a file with a name into a multipart endpoint so that we can use that as a reference in the future